python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/community_lists.schema.yml

```diff
@@ -8,8 +8,14 @@ type: dict
 keys:
   community_lists:
     type: list
+    description: If both `community_lists` and `ip_community_lists` are defined, then `ip_community_lists` takes precedence.
     primary_key: name
     display_name: Community Lists (legacy model)
+    deprecation:
+      warning: true
+      removed: false
+      remove_in_version: 6.0.0
+      new_key: ip_community_lists
```

Comment:
remove the below part `items`

Reply:
We are not removing the key. So we need the items.